src/explorer/installation/installation-explorer.ts

```diff
@@ -0,0 +1,100 @@
+import * as vscode from 'vscode';
+import { CommandResult } from '../../commands/result';
+
+import * as porter from '../../porter/porter';
+import { Installation, InstallationHistoryEntry } from '../../porter/porter.objectmodel';
+import { Shell } from '../../utils/shell';
+import { viewLogs } from '../../views/logs';
+import { viewOutputs } from '../../views/outputs';
+import { ExplorerErrorNode } from '../errornode';
+import { Node } from '../node';
+
+export class InstallationExplorer implements vscode.TreeDataProvider<InstallationExplorerTreeNode> {
```

Comment:
Is there a way to trigger a refresh of a node? So that if I have it open and then execute another porter command, be able to refresh and see the action and its logs?

Reply:
I haven't implemented that but yes, it probably needs to be part of this PR.